manticore/platforms/linux.py

```diff
@@ -1910,7 +1921,7 @@ def sys_acct(self, path):
         return -1
 
     def sys_exit(self, error_code):
-        'Wrapper for sys_exit_group'
+        '''Wrapper for sys_exit_group'''
```

Comment:
```suggestion
        """Wrapper for sys_exit_group"""
```
It might be good to add a sentence here why is exit a wrapper for exit_group - is it that we lack some functionality here? Does it work the same in kernel and it is a technical debt from the past?